google/api_core/grpc_helpers.py

```diff
@@ -210,21 +216,59 @@ def _create_composite_credentials(
         )
 
     if credentials_file:
-        credentials, _ = google.auth.load_credentials_from_file(credentials_file, scopes=scopes)
+        try:
+            credentials, _ = google.auth.load_credentials_from_file(
+                credentials_file,
+                scopes=scopes,
+                default_scopes=default_scopes
+            )
+        # google-auth < x.x.x does not have `default_scopes`
+        # TODO: remove this try/except once google-auth >= x.x.x is required
+        except TypeError:
+            credentials, _ = google.auth.load_credentials_from_file(
+                credentials_file,
+                scopes=scopes or default_scopes,
+            )
```

Comment:
`x.x.x` as a place-holder. It will be updated with the real version once google-auth has been released.

We'll be able to test both branches for coverage since unit-3.6 will install a version *without* `default_scopes` and the other unit test runs will install versions *with* `default_scopes`. 

When this is merged, an issue will be created to track removing these try/excepts.

CC @tswast

Reply:
Thank you!

Reply:
You might want to actually parse the versions, like we do in pandas-gbq. https://github.com/pydata/pandas-gbq/blob/81ab6b8f01c5a2cd1033e687a3d8bb7fe3f43c98/pandas_gbq/gbq.py#L61-L63

There are several options for this, the pandas-gbq library uses an old package from distutils/setuptools. It may be more reliable to use this package: https://packaging.pypa.io/en/latest/version.html#packaging.version.parse

Reply:
Ah that looks like it will be cleaner. Will do!